cluster/javadsl/src/main/resources/reference.conf

```diff
@@ -1 +1,2 @@
 play.modules.enabled += com.lightbend.lagom.internal.javadsl.cluster.JoinClusterModule
+play.modules.enabled += com.lightbend.lagom.internal.javadsl.cluster.typed.ClusterShardingTypedModule
```

Comment:
If you had `lagom-cluster` in the dependency tree, you now have an instance of Akka Cluster Sharding Typed in Guice.